src/cpp/session/SessionModuleContext.cpp

```diff
@@ -1070,6 +1070,25 @@ FilePath findProgram(const std::string& name)
    }
 }
 
+bool addTinytexToPathIfNecessary()
+{
+   if (isPdfLatexInstalled())
+      return false;
+
+   FilePath binPath;
+
+#if defined(_WIN32)
+   FilePath appData(core::system::getenv("APPDATA"));
+   binPath = appData.complete("TinyTeX\\bin\\win32");
+#elif defined(__APPLE__)
+   binPath = resolveAliasedPath("~/Library/TinyTeX/bin/x86_64-darwin");
```

Comment:
Any chance we could ask if the tinytex package itself where it installed pdflatex? (worried that these paths could drift in the future with updates to tinytex)

Reply:
That's a good idea. I think we still want this fallback for the case where the tinytex package is not installed or not on the library paths for some reason, though. I'll make a change.